Framework/MDAlgorithms/src/IntegrateEllipsoids.cpp

```diff
@@ -483,14 +483,13 @@ void IntegrateEllipsoids::exec() {
   std::vector<double> principalaxis1, principalaxis2, principalaxis3;
   std::vector<double> sateprincipalaxis1, sateprincipalaxis2,
       sateprincipalaxis3;
-  V3D peak_q;
   for (size_t i = 0; i < n_peaks; i++) {
     V3D hkl(peaks[i].getIntHKL());
     V3D mnp(peaks[i].getIntMNP());
 
     if (Geometry::IndexingUtils::ValidIndex(hkl, 1.0) ||
         Geometry::IndexingUtils::ValidIndex(mnp, 1.0)) {
-      peak_q = peaks[i].getQLabFrame();
+      V3D peak_q = peaks[i].getQLabFrame();
```

Comment:
```suggestion
      const V3D peak_q = peaks[i].getQLabFrame();
```

Reply:
I `const`ed this and did some general refactoring in the loop.